core/src/ast_builder/expr/aggregate.rs

```diff
@@ -1,25 +1,76 @@
 use {
     super::ExprNode,
     crate::{
-        ast::{Aggregate, Expr},
+        ast::{Aggregate, CountArgExpr, Expr, ToSql},
+        parse_sql::parse_expr,
         result::{Error, Result},
+        translate::translate_expr,
     },
 };
 
 #[derive(Clone)]
 pub enum AggregateNode {
+    Count(CountExprNode),
     Sum(ExprNode),
     Min(ExprNode),
     Max(ExprNode),
     Avg(ExprNode),
     Variance(ExprNode),
 }
 
+#[derive(Clone)]
+pub enum CountExprNode {
+    Text(String),
+}
```

Comment:
You can consider to have one more enum type
```rust
pub enum CountExprNode {
    Text(String),
    Expr(ExprNode),
}
```
Then, we don't need to use `to_sql` conversion